carbonmark/components/pages/Projects/Banners/BankTransfer.tsx

```diff
@@ -0,0 +1,85 @@
+import { t } from "@lingui/macro";
+import { ButtonPrimary } from "components/Buttons/ButtonPrimary";
+import { ButtonSecondary } from "components/Buttons/ButtonSecondary";
+import FeatureBanner from "components/FeatureBanner";
+import { urls } from "lib/constants";
+import { FC, useEffect, useState } from "react";
+
+export const BankTransferBanner: FC = () => {
+  const [isVisible, setIsVisible] = useState(false);
+  const [localStorage, setLocalStorage] = useState("0");
```

Comment:
Maybe we can introduce a type here? It will be easier to work with. Something like:

```suggestion
type BannerState = "not shown yet" | "do not show" | "some other state" 
…
  const [localStorage, setLocalStorage] = useState<BannerState>("not shown yet");
```

Reply:
@sprrwhwk Yeah this is a nice suggestion and will make it easier for anyone else looking at the code to understand each state.